var/spack/repos/builtin/packages/delphes/package.py

```diff
@@ -62,4 +62,4 @@ def cmake_args(self):
 
     def setup_run_environment(self, env):
         # make the cards distributed with delphes more easily accessible
```

Comment:
Does this comment still apply?

Reply:
yes, it still makes sense, since you can access them with `$DELPHES_DIR/cards/`